src/main/java/com/diffblue/demo/ecommerce/forms/AddressForm.java

```diff
@@ -20,7 +20,6 @@
   @NotEmpty
   private String postcode;
 
-
```

Comment:
Does this file only have whitespace changes? It would make sense to remove the change from the PR.